client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -15,20 +15,30 @@ import designs from './available-designs.json';
 import { usePath, Step } from '../../path';
 import { isEnabled } from '../../../../config';
 import Link from '../../components/link';
+import { SubTitle, Title } from '../../components/titles';
+
 import './style.scss';
-import { SubTitle, Title } from 'landing/gutenboarding/components/titles';
 
-const DesignSelector: FunctionComponent = () => {
+// Values for springs:
+const ZOOM_OFF = { transform: 'scale(1)' };
+const ZOOM_ON = { transform: 'scale(1.03)' };
+const SHADOW_OFF = { boxShadow: '0 0 0px rgba(0,0,0,.2)' };
+const SHADOW_ON = { boxShadow: '0 0 15px rgba(0,0,0,.2)' };
+
+const DesignSelector: React.FunctionComponent = () => {
 	const { __: NO__ } = useI18n();
 	const { push } = useHistory();
 	const makePath = usePath();
-	const { selectedDesign } = useSelect( select => select( ONBOARD_STORE ).getState() );
 	const { setSelectedDesign, resetOnboardStore } = useDispatch( ONBOARD_STORE );
 
 	const handleStartOverButtonClick = () => {
 		resetOnboardStore();
 	};
 
+	// Track hover/focus
+	const [ hoverDesigns, setHoverDesigns ] = React.useState< Set< string > >( new Set() );
+	const [ focusDesigns, setFocusDesigns ] = React.useState< Set< string > >( new Set() );
```

Comment:
Multiple designs can be both hovered over, and focused? :thinking: 

Also, curious why you chose `Set` over a plain ol' array `[]` :slightly_smiling_face:

Reply:
>Multiple designs can be both hovered over, and focused? 🤔

Was that for considering for keyboard navigation? The hover state should prolly not be activey for anything whenever someone is keyboard navigating?

Reply:
> Multiple designs can be both hovered over, and focused? 🤔

🤔 We could probably use a single item, but there's nothing inherently wrong with a singleton set. I'll see if that makes sense. We can be more aggressive on focus/mouseenter (just set), but will need to be more defensive on blur/mouseout (set `null` iff current is the blurred/mouseleft element).

> Also, curious why you chose Set over a plain ol' array [] 🙂

`Set`s is nice for unordered collections with unique elements, and is ideal for asking "is this in the set?" which is what we want to do here.

> Was that for considering for keyboard navigation? The hover state should prolly not be activey for anything whenever someone is keyboard navigating?

Yes, if we don't apply hover style, what's the visual indicator for keyboard navigation?